scripts/dockerfiles/Dockerfile.main-debug-base

```diff
@@ -0,0 +1,71 @@
+FROM amazon/aws-for-fluent-bit:build as builder
+COPY ./scripts/dockerfiles/Dockerfile.build /Dockerfile.1.build
```

Comment:
why are we copying one dockerfile inside another??

Reply:
Just for documentation on what else the image is based on... I can get rid of this if you want